cli/utilities/containers.py

```diff
@@ -33,11 +30,89 @@ def login(
         elif username and password:
             cmd += f" --username {username} --password {password}"
         else:
-            raise ContainerRegistryError("Authentication details needs to be provided")
```

Comment:
Don't include this here, will have separate PR for this change

Reply:
Moved out of this PR

Reply:
????
Still changes are here